src/model/User.model.js

```diff
@@ -1,24 +1,35 @@
 export default class User {
-  constructor (username, groups, created, admin) {
+  constructor (username, groups, created, admin, server) {
     this.username = username
     this.groups = groups
     this.created = created
     this.admin = admin
+    /**
+     * @type {string} - server URL
+     */
+    this.server = server
   }
 
-  getUserName () {
-    return this.username
-  }
-
-  getGroups () {
-    return this.groups
-  }
-
-  getCreated () {
-    return this.created
-  }
-
-  isAdmin () {
-    return this.admin
```

Comment:
These methods are simply my Java-side speaking louder, sorry :+1:  not necessary/useful, as there's no easy way to have private members/encapsulation in JS/ES6 (there are some ways... but they are not simple.. once we move to typescript this will be a breeze to implement).